internals/plan/plan.go

```diff
@@ -154,6 +155,9 @@ func (s *Service) Merge(other *Service) {
 	if other.Group != "" {
 		s.Group = other.Group
 	}
+	if other.WorkingDir != "" {
+		s.WorkingDir = other.WorkingDir
+	}
```

Comment:
Presumably you need to add something to the `Service.Copy` method too.

Reply:
and maybe add a plan test to check this parses correctly?

Reply:
You don't actually need it for `Service.Copy` because the value fields are copied by [this line](https://github.com/canonical/pebble/blob/8902cbe618e4b1bcfbc16a2a6c597c6c49aa88c7/internals/plan/plan.go#L98).

Reply:
> maybe add a plan test to check this parses correctly?

Good *plan*. I'll do that tomorrow.